pkg/pi/pi.go

```diff
@@ -161,3 +164,24 @@ func (p *Pi) openEtcd() *Pi {
 	p.etcd = e
 	return p
 }
+
+func (p *Pi) GetBasicCfg() *config.BasicConfig {
+	p.cfg = config.LoadConf()
+	basicConfig := &config.BasicConfig{
+		PlatformName: p.cfg.Basic.PlatformName,
+		PlatformUrl:  p.cfg.Basic.PlatformUrl,
+	}
+	return basicConfig
+}
+
+func (p *Pi) SetBasicCfg(basicConfig config.BasicConfig) error {
```

Comment:
We often use pointer here
func (p *Pi) SetBasicCfg(basicConfig *config.BasicConfig) error {